health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/preparing/PreparingExerciseScreen.kt

```diff
@@ -173,81 +167,88 @@ fun PreparingExerciseScreen(
 ) {
     val location = (uiState as? PreparingScreenState.Preparing)?.locationAvailability
 
-    val scrollState = rememberScrollState()
+    val columnState = rememberResponsiveColumnState(
+        contentPadding = ScalingLazyColumnDefaults.padding(
+            first = ItemType.Unspecified, last = ItemType.Unspecified
+        )
+    )
 
-    ScreenScaffold(scrollState = scrollState) {
-        ScalingLazyColumnDefaults.padding(
-            first = ItemType.Unspecified,
-            last = ItemType.Unspecified
-        )()
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .background(MaterialTheme.colors.background),
-            verticalArrangement = Arrangement.Center
+    ScreenScaffold(scrollState = columnState) {
+        ScalingLazyColumn(
+            columnState = columnState
         ) {
-            Row(
-                modifier = Modifier.height(25.dp)
-            ) {
-                Text(
-                    textAlign = TextAlign.Center,
-                    maxLines = 1,
-                    overflow = TextOverflow.Ellipsis,
-                    text = stringResource(id = R.string.preparing_exercise),
+            item {
+
+                Column(
```

Comment:
putting a Column in an item isn't typical, why not remove column and have each one as an item?

Reply:
Done! Thanks